app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -233,8 +233,10 @@ def self.host_inv_to_ip(inv, hostname = nil)
       require 'ipaddr'
       default_gw = IPAddr.new(default_gw)
 
-      vnics   = inv.fetch_path("config", "network", "consoleVnic")
-      vnics ||= inv.fetch_path("config", "network", "vnic")
+      vnics         = inv.fetch_path("config", "network", "vnic")
+      console_vnics = inv.fetch_path("config", "network", "consoleVnic")
+      vnics.concat(console_vnics) unless console_vnics.nil?
```

Comment:
- I think we did the consoleVnics first because it was more likely to contain the value we cared about.  Should that still be first?
- If there are no nics at all on the VM, what happens?  Do you get vnics set to nil, and thus concat blows up?
- Wondering if this might be cleaner/safer:

```ruby
vnics = inv.fetch_path("config", "network", "consoleVnic").to_miq_a + inv.fetch_path("config", "network", "vnic").to_miq_a
```